cincinnati/src/plugins/internal/arch_filter.rs

```diff
@@ -99,10 +99,13 @@ fn infer_version(
             Ok(default_arch)
         }
         (None, None) => Err(GraphError::MissingParams(
-            vec!["arch", "version"]
-                .into_iter()
-                .map(String::from)
-                .collect(),
+            vec![&format!(
+                "arch or version=<{}",
+                default_arch_threshold_version
```

Comment:
@wking I'm not sure if we want to include the threshold in the error message

Reply:
this is about folks updating their callers, so I think it's fine to just say "set `arch`".  Our supporting `version=4.1.z` as an alternative can be an undocumented internal workaround.